site/homepage/layouts/partials/head.html

```diff
@@ -3,7 +3,7 @@
   <meta name="robots" content="all,follow">
   <meta name="googlebot" content="index,follow,snippet,archive">
   <meta name="viewport" content="width=device-width, initial-scale=1">
-  <title>{{ .Title }}</title>
+  <title>{{ .Title }} :: {{ .Site.Title }}</title>
```

Comment:
is this related to the twitter stuff?

Reply:
No. I consider this a general improvement, because it unifies the way how the title is build between homepage and documentation. But it should have been a separate commit/PR.

Reply:
exactly :+1:
can you please create a separate PR for it?

Reply:
I removed the title configuration from this PR.